src/Gui/View3DInventor.cpp

```diff
@@ -145,7 +145,6 @@ View3DInventor::View3DInventor(Gui::Document* pcDocument, QWidget* parent,
     OnChange(*hGrp,"UseBackgroundColorMid");
     OnChange(*hGrp,"ShowFPS");
     OnChange(*hGrp,"ShowNaviCube");
-    OnChange(*hGrp,"CornerNaviCube");

```

Comment:
This causes a regression because it's not possible any more to effectively change the corner. The NaviCube will always stay at the top right corner.

Two changes are needed to make it work again:
1. Get access to the NaviCube parameter group, attach the viewer to it and call `OnChange(*hNaviGrp,"CornerNaviCube");`